module/skill/QuinticWalk/src/QuinticWalk.cpp

```diff
@@ -252,6 +270,11 @@ namespace module::skill {
                     emit(std::make_unique<Stability>(Stability::STANDING));
                 }
                 calculate_joint_goals();
+
+                auto walking_state          = std::make_unique<WalkingState>();
+                walking_state->is_walking   = false;
+                walking_state->walk_command = Eigen::Vector3f::Zero();
+                emit(std::move(walking_state));
```

Comment:
unless you particularly like it for readability this could be one line
`emit(std::make_unique<WalkingState>(false, Eigen::Vector3f::Zero());`